src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -410,13 +418,28 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
             // Set limit to the query for pagination support
             metadataQueryClone.limit = DEFAULT_PAGE_SIZE;
         }
+
+        if (sortBy && sortDirection) {
+            metadataQueryClone.order_by = [
+                ...(metadataQueryClone.order_by ?? []),
+                {
+                    field_key: sortBy,
+                    direction: sortDirection,
+                },
```

Comment:
> ...(metadataQueryClone.order_by ?? []),

does BoxApps allow for chained sorting? isn't this a little confusing from the user's perspective + does this actually work as we expect it to work?